main.go

```diff
@@ -331,3 +341,36 @@ func initKubeConfig(kcLocation string) *rest.Config {
 
 	return kubeConfig
 }
+
+func initTransport(upstreamCAFile string) (http.RoundTripper, error) {
+	if upstreamCAFile == "" {
+		return nil, nil
```

Comment:
if there is no upstream CA file here, we are returning `nil` here, which effectively uses the default transport later on. For brevity, and because this is an init method, let's return the default transport here to make it explicit.